### ISSUE TITLE
Feature: In Blog page make the links directing to the actual blogs. 

### ISSUE THREAD

**Issue by The-Shivam-garg** [The-Shivam-garg/BigB-E-learn-Websit-e#34]
![image](https://user-images.githubusercontent.com/83035995/152298847-9941628e-ced5-45e2-80b4-e62375e9dd2b.png)

Make the **Read more** button linkable to the blogs.

**Comment by RajulPE** [comment]
I am interested in working on this issue under JWOC.

**Comment by The-Shivam-garg** [comment]
Thank you for showing your interest in this issue . I'm assigning this issue to you @RajulPE 
Best wishes for your further contribution in open-source.😊

**Comment by RajulPE** [comment]
Thank You @The-Shivam-garg

**Comment by sayan404** [comment]
I want work in this issue under JWOC.

**Comment by The-Shivam-garg** [comment]
yeah ! @sayan404 you can work on this site, this is my hashnode profile here you will get the blogs that are mentioned on the site. Do link them with my hashnode blogs https://hashnode.com/@Shiva12

**Comment by sayan404** [comment]
Hi @The-Shivam-garg , I am not getting all the links of the courses mentioned  in the web page . What should I do ?

**Comment by The-Shivam-garg** [comment]
hey @sayan404 , just search over google and link any relatable blog with it.

**Comment by sayan404** [comment]
ok @The-Shivam-garg  ... although i have made a pr . i will merge the links and will update the pr .. is it ok ?

**Comment by sayan404** [comment]
@The-Shivam-garg  I have made  an another PR .
